roamance/src/components/profile/profile-header.tsx

```
@@ -230,157 +237,246 @@ export function ProfileHeader({
-                    <Calendar className="h-3.5 w-3.5" />
+                    {/* Adjusted icon size */}
+                    <Calendar className="h-3 w-3 sm:h-3.5 sm:w-3.5 group-hover:animate-pulse" />
                     <span className="font-medium">
                       Joined{' '}
                       {`${getMonthName(getDateParts(`${user?.created_at}`).month)} ${getDateParts(`${user?.created_at}`).year}` ||
                         'April 20XX'}
                     </span>
                   </Badge>
-                </div>
+                  {/* Adjusted decorative element size */}
+                  <div className="absolute -bottom-1 -right-1 w-6 h-6 sm:w-8 sm:h-8 bg-gradient-to-tr from-ocean/30 to-transparent rounded-full blur-lg opacity-70"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)